fuzz/fuzz_targets/simple_model.rs

```diff
@@ -7,17 +7,16 @@
 #![no_main]
 use libfuzzer_sys::fuzz_target;
 use parity_db_fuzz::*;
-use std::{
-	collections::{BTreeMap, BTreeSet, HashMap},
-	path::Path,
-};
-
-#[derive(Default)]
-struct Model {
-	current: BTreeMap<u8, u8>,
-	removed: BTreeSet<u8>,
+use std::{collections::HashMap, path::Path};
+
+#[derive(Clone, Debug)]
+struct Layer {
+	values: [Option<Option<u8>>; 256],
```

Comment:
Quite a few 255/256 values all over the code. Lets make it constant or use something like `model.values.len()` wherever possible, instead of hardcoding it.

Reply:
Done